README.md

```diff
@@ -152,6 +152,30 @@ client = Kubeclient::Client.new(
 
 You can find information about tokens in [this guide](http://kubernetes.io/docs/user-guide/accessing-the-cluster/) and in [this reference](http://kubernetes.io/docs/admin/authentication/).
 
+#### Google's Application Default Credentials
+
+On Google Compute Engine, Google App Engine, or Google Cloud Functions, as well as `gcloud`-connfigured systems
+with [application default credentials](https://developers.google.com/identity/protocols/application-default-credentials), 
+you can use the token provider to authorize `kubeclient`.
+
+This requires the [`googleauth` gem](https://github.com/google/google-auth-library-ruby) that is _not_ included in 
+`kubeclient` dependencies so you should add it to your bundle.
+
+```ruby
+require 'googleauth'
```

Comment:
this is not strictly neccessary here, just to emphasize the dependency on the gem, right?

Reply:
I'd rather not assume bundler does `require`.  Some people practice `require: false` in Gemfile.  And I suppose some people [citation needed ;-] use `gem install` outside bundler.
=> if we don't require in the code, keep this require in README.

Personally I'd do `require` on demand inside the `token` method.  Because it *is* a run time dependency of this code, only not a bundler dependency.  So user is responsible to install it, but if installed we can use it.
But I don't care much, your call.

IMHO LoadError is clear & actionable enough, tells you what gem you need.
(But an uncatched NameError would be too obscure to my taste.)
If you want a rescue for a friendlier message, that's OK too...

Reply:
Sounds good. I'll leave it as is. It accomplishes the job and I don't want to add more code.